src/ensembl/src/shared/image-button/ImageButton.scss

```diff
@@ -1,29 +1,59 @@
 @import 'src/styles/common';
 
 .default {
+  position: relative;
   svg {
     width: 100%;
     height: 100%;
     fill: $ens-blue;
     cursor: pointer;
+    transition: opacity 0.4s;
+    -webkit-transition: opacity 0.4s;
```

Comment:
Here and elsewhere: won’t `postcss-preset-env` that we have in our dependencies add all this stuff automatically?

Reply:
Yes it should

Reply:
Cool, I wasn't aware of that. I've updated it now.